edit/buildozer_test.go

```diff
@@ -114,3 +119,69 @@ func TestCmdRemoveComment(t *testing.T) {
 		}
 	}
 }
+
+type targetExpressionToBuildFilesTestCase struct {
+	rootDir, target string
+	buildFiles      []string
+}
+
+func runTestTargetExpressionToBuildFiles(t *testing.T, buildFileName string) {
+	tmp, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmp)
+	if err := os.MkdirAll(filepath.Join(tmp, "a", "b"), 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := os.MkdirAll(filepath.Join(tmp, "a", "c"), 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := ioutil.WriteFile(filepath.Join(tmp, "WORKSPACE"), nil, 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := ioutil.WriteFile(filepath.Join(tmp, buildFileName), nil, 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := ioutil.WriteFile(filepath.Join(tmp, "a", buildFileName), nil, 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := ioutil.WriteFile(filepath.Join(tmp, "a", "b", buildFileName), nil, 0755); err != nil {
+		t.Fatal(err)
+	}
+	if err := ioutil.WriteFile(filepath.Join(tmp, "a", "c", buildFileName), nil, 0755); err != nil {
+		t.Fatal(err)
+	}
+
+	for _, tc := range []targetExpressionToBuildFilesTestCase{
+		{tmp, "//", []string{filepath.Join(tmp, buildFileName)}},
+		{tmp, "//:foo", []string{filepath.Join(tmp, buildFileName)}},
+		{tmp, "//a", []string{filepath.Join(tmp, "a", buildFileName)}},
+		{tmp, "//a:foo", []string{filepath.Join(tmp, "a", buildFileName)}},
+		{tmp, "//a/b", []string{filepath.Join(tmp, "a", "b", buildFileName)}},
+		{tmp, "//a/b:foo", []string{filepath.Join(tmp, "a", "b", buildFileName)}},
+		{tmp, "//...", []string{filepath.Join(tmp, buildFileName), filepath.Join(tmp, "a", buildFileName), filepath.Join(tmp, "a", "b", buildFileName), filepath.Join(tmp, "a", "c", buildFileName)}},
+		{tmp, "//a/...", []string{filepath.Join(tmp, "a", buildFileName), filepath.Join(tmp, "a", "b", buildFileName), filepath.Join(tmp, "a", "c", buildFileName)}},
+		{tmp, "//a/b/...", []string{filepath.Join(tmp, "a", "b", buildFileName)}},
+		{tmp, "//a/c/...", []string{filepath.Join(tmp, "a", "c", buildFileName)}},
+		{tmp, "//a/c/...:foo", []string{filepath.Join(tmp, "a", "c", buildFileName)}},
+	} {
+		buildFiles := targetExpressionToBuildFiles(tc.rootDir, tc.target)
+		expectedBuildFilesMap := make(map[string]string)
```

Comment:
I'm not sure if this is the right way to do this comparison. (I only learned Go this weekend to fix the `BUILD.bazel` bug, so my code might not be that idiomatic...)